packages/namekit-react/src/components/EnsAvatar.tsx

```diff
@@ -60,7 +60,9 @@ export const EnsAvatar = ({ name }: AvatarProps) => {
 
   const EnsAvatarLayout =
     name && name.normalization !== Normalization.normalized ? (
-      <NotNormalizedNameEnsAvatar normalization={name.normalization} />
+      <NotNormalizedNameEnsAvatar
+        normalization={name.normalization as Normalization}
```

Comment:
Hmm. Why wouldn't the type system already know that `name.normalization` is type `Normalization`?